### ISSUE TITLE
No ability to build dev changes

### ISSUE THREAD

**Issue by wtrocki** [aerogear/ios-showcase-template#18]
it's not possible to merge any PR now that has dev changes. We need to include SDK into the build the same as we have done for JS SDK:

https://github.com/aerogear/cordova-showcase-template/pull/26

Context: https://github.com/aerogear/ios-showcase-template/pull/16

**Comment by wtrocki** [comment]
@wei-lee Is that ok with you?

**Comment by wei-lee** [comment]
@wtrocki sounds good to me

**Comment by wei-lee** [comment]
@wtrocki actually, I just noticed that in aerogear/cordova-showcase-template#26, if there are new changes in the SDK as well as in the show case app, the SDK changes has to be merged to master first in order to build the show case app, is that the expected behaviour?

**Comment by wtrocki** [comment]
@wei-lee 
I have tested many approaches in the past and the best one is to have showcase app in the same repo. If there needs to be separate repo master worked well in the past for me. 

We cannot merge change for showcase application without SDK being reviewed and merged. Verification is happening with linked sources so there is no strict blocker.

**Comment by wtrocki** [comment]
TL;DR - This works great with current dev process but we can always improve it.

**Comment by wei-lee** [comment]
@wtrocki yes, you are right. This is the best way to do it.

**Comment by wtrocki** [comment]
Taking to unblock @StephenCoady

**Comment by wtrocki** [comment]
@jhellar I will need to change `LOCAL=true pod install` to `LOCATION=../ pod install` as it many different cases location can be different different. This will help our scripts to be much simpler. I will also adjust docs.

**Comment by aidenkeating** [comment]
@wtrocki Is this ok to close?
